### PR TITLE
BaSP-TG-37: List admins page

### PR DESCRIPTION
**Pull request by Facund0Fernandez**

### CODE REVIEW

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -1,11 +1,45 @@
 import styles from './admins.module.css';
+import { useState, useEffect } from 'react';
+import List from './List/AdminsList';
+
+const Admins = () => {
+  const URL = `${process.env.REACT_APP_API_URL}/admins`;
+  const [list, setList] = useState([]);
+
+  // API request to get admins
```
Comment (HiginiaMed) [inline src/Components/Admins/index.js:9]: Remember to delete the comments!

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -1,11 +1,45 @@
...
+      const response = await fetch(URL, { method: 'GET' });
+      const responseJson = await response.json();
+      setList(responseJson.data);
+      console.log('data', responseJson.data);
+    } catch (error) {
+      console.error(error);
+    }
+  }, []);
+
+  //API request to delete admins
```
Comment (HiginiaMed) [inline src/Components/Admins/index.js:21]: Here delete comments to

File: src/Components/Admins/List/AdminsList.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import AdminsItem from '../ListItem/AdminsItem';
+import './list.module.css';
+
+const List = ({ list, deleteAdmin }) => {
+  console.log('admins list', list);
```
Comment (Gearyandres94) [inline src/Components/Admins/List/AdminsList.js:6]: delete console.log

File: src/Components/Admins/ListItem/AdminsItem.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styles from './adminsItem.module.css';
+
+function AdminsItem({ listItem, deleteAdmin }) {
+  console.log('props', listItem);
```
Comment (Gearyandres94) [inline src/Components/Admins/ListItem/AdminsItem.js:5]: delete this console.log too

File: src/Components/Admins/index.js [hunk 5]
```diff
@@ -1,11 +1,45 @@
...
+  const URL = `${process.env.REACT_APP_API_URL}/admins`;
+  const [list, setList] = useState([]);
+
+  // API request to get admins
+  useEffect(async () => {
+    try {
+      const response = await fetch(URL, { method: 'GET' });
+      const responseJson = await response.json();
+      setList(responseJson.data);
+      console.log('data', responseJson.data);
```
Comment (Gearyandres94) [inline src/Components/Admins/index.js:15]: Delete this console.log too.

File: src/Components/Admins/index.js [hunk 6]
```diff
@@ -1,11 +1,45 @@
...
+    } catch (error) {
+      console.error(error);
+    }
+  }, []);
+
+  //API request to delete admins
+  const deleteAdmin = async (_id) => {
+    const response = await fetch(`${URL}/${_id}`, { method: 'DELETE' });
+    const responseJson = await response.json();
+    console.log(responseJson);
```
Comment (Gearyandres94) [inline src/Components/Admins/index.js:25]: This one should be deleted.

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]

**Review by GuidoCerioni — APPROVED** [review]
Nice work Facu!

**Review by CarVigil — APPROVED** [review]
Everything works fine! Excelent 👌

**Comment by tobiaspujol** [comment]
Works fine!

**Review by Gearyandres94 — APPROVED** [review]
Works fine!! great job!!

**Pull request merged**